### ISSUE TITLE
Quiver Feature

### ISSUE THREAD

**Issue by lostopkk** [MehVahdJukaar/Supplementaries#844]
### Before continuing, do the following checklist:

- [X] Read the list of features via [Home](https://github.com/MehVahdJukaar/Supplementaries/wiki)
- [X] Confirm there is no existing feature that meets your ideal feature
- [X] Confirm there is no existing submitted feature request in the issue

## Description:
Quiver Feature

## Why should we implement the feature?
When quiver is equipped, every time you pick up a arrow it automatically adds it to the quiver instead of the inventory.

**Comment by MehVahdJukaar** [comment]
Hm I thought it did that. Maybe it was just on arrows picked up when as entities

**Comment by lostopkk** [comment]
> Hm I thought it did that. Maybe it was just on arrows picked up when as entities

While I was playing this didn't happen, it always went to my inventory and ended up being a slot that was taken for no reason since I had the quiver equipped...

**Comment by MehVahdJukaar** [comment]
and this was on forge 1.20?

**Comment by lostopkk** [comment]
> and this was on forge 1.20?

fabric 1.20.1

**Comment by MehVahdJukaar** [comment]
yeah those have not beed added to fabric version as it lacks an API to do so
